UnityProject/Assets/Editor/LevelBuilder/Util/GUIUtils.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
can we check if all these are necessary? :)

Reply:
I am quite sure that in most files we have unnecessary imports... I opened #65